docs/contributing/contributing.md

```diff
@@ -3,34 +3,76 @@ id: contributing
 
- This document describes the main flow that covers most of the cases and summarizes everything you must do before and after opening a Pull Request on GitHub to contribute your pack.
+This guide doesn't cover all the topics: please browse the left sidebar and use the search bar to find what you need, and reach out for help over [Slack](https://start.paloaltonetworks.com/join-our-slack-community) on the `#demisto-developers` channel when in doubt.
 
-Before proceeding, sure to check the content contribution [checklist](../contributing/checklist) to make sure you have created everything you need.
 
 ## Before opening a Pull Request
 
 In order to be able to submit a Pull Request to the Cortex XSOAR [GitHub Repository](https://github.com/demisto/content), you need to:
 
-- Have a [GitHub](https://github.com) account that you'll use to create the fork and open the Pull Request.
-- Create a fork of the repo in your own Github account. You can follow the instruction described [GitHub docs](https://guides.github.com/activities/forking/) and use the examples from the [Development environment tutorial](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-2-fork-the-github-repo).
-- Create a new branch that will contain all of your contributed content. You can reference the [Create a branch](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-6-create-a-branch) step in the development setup toturial. 
+- Make sure to check the content contribution [checklist](../contributing/checklist) to make sure you have created everything you need.
+- Make sure you follow the general [prerequisites](getting-started-guide#before-you-start), satisfy the [requirements](../contributing/contrib-requirements) and set up the [development environment](../concepts/dev-setup) ([tutorial](../tutorials/tut-setup-dev)).
+- Make sure you are working on a GitHub **fork** of the XSOAR content repository, and **create a branch** for your contribution (do **NOT** work on *master*).
 - If you're an XSOAR partner, have your `partner-id` (this should have been communicated to you over the onboarding emails from the Alliance team).
-- Join our our [Slack DFIR Community](https://www.demisto.com/community/), this is the place to ask questions, consult about use-case and raise issues regarding the Cortex XSOAR platform (use the `#demisto-developers` channel).
 - Design Document: we encourage you to prepare a Design Document that describes the capabilities of your Pack. Usually it's a Google Doc shared with you by the Alliances team. Click [here](https://docs.google.com/document/d/1wETtBEKg37PHNU8tYeB56M1LE314ux086z3HFeF_cX0) to see a compiled example of a Design Document.
```

Comment:
This should also be moved to guidelines IMO

Reply:
done